cmd/harborAPI.go

```diff
@@ -129,6 +130,26 @@ func update(token string, url string, data interface{}) {
 	}
 }
 
+// GetLogs returns a string of all container logs for a shipment
+func GetLogs(barge string, shipment string, env string) string {
+    var url string = helmitURI + "/harbor/" + barge + "/" + shipment + "/" + env
```

Comment:
Should we use the 'uritemplates' library to be consistent with the other functions in this file?